.devcontainer/library-scripts/docker-in-docker-debian.sh

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env bash
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+#
+# Docs: https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/docs/docker-in-docker.md
+# Maintainer: The VS Code and Codespaces Teams
+#
+# Syntax: ./docker-in-docker-debian.sh [enable non-root docker access flag] [non-root user] [use moby]
```

Comment:
>docker-in-docker-debian [](http://example.com/codeflow?start=12&length=23)

is this file any different from the ones we have int he other tools devcontainers?

Reply:
it is identical